src/shippingservice/Cargo.toml

```diff
@@ -21,8 +21,8 @@ prost-types = "0.12"
 tokio = { version = "1.36", features = ["macros", "rt-multi-thread"] }
 opentelemetry = { version = "0.22", features = ["trace"] }
 opentelemetry_sdk = { version = "0.22", features = ["rt-tokio"]}
-opentelemetry-semantic-conventions = "0.14.0"
-opentelemetry-http = "0.11.0"
+opentelemetry-semantic-conventions = "0.15.0"
+opentelemetry-http = "0.12.0"
```

Comment:
`opentelemetry-http` depends on `opentelemetry="0.23"`, and brings it as transient dependency.
This will conflict with the current `0.22` we are using.

Ideally we could simply update the `opentelemetry` to `0.23`, but we use `reqwest-tracing` that doesn't have an updated crate using `0.23` yet.

We need to keep it on `0.11.0` for now.